selenium/src/test/java/com/deque/html/axecore/selenium/Axe43xIntegrationTest.java

```diff
@@ -544,6 +560,7 @@ private void initDriver(String browser) {
       ChromeOptions options = new ChromeOptions();
       options.setUnhandledPromptBehaviour(UnexpectedAlertBehaviour.ACCEPT);
       options.addArguments(
+          "--remote-allow-origins=*",
```

Comment:
👀  did you mean to add this?

Reply:
Update: 

This appears to be an issue with [Selenium 4.4](https://github.com/SeleniumHQ/selenium/issues/11750) and [Chromedriver](https://bugs.chromium.org/p/chromium/issues/detail?id=1422444), we can likely now as of Selenium > 4.5 remove this as part of the test